### PR TITLE
Add install build dependencies script

### PR DESCRIPTION
**Pull request by dati91**

### CODE REVIEW

**Comment by szledan** [comment]
Why don't you use python for the script?
Please check the solutions of @dbatyai in `tools/scripts`.

**Comment by szledan** [comment]
Please close #34 issue if it's merged.

File: tools/scripts/install-build-deps.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Copyright (C) 2015-2016, Gepard Graphics
+# Copyright (C) 2015, Attila Dusnoki <dati91@gmail.com>
```
Comment (szledan) [inline tools/scripts/install-build-deps.sh]: Please change 2015 to 2016.

File: tools/scripts/install-build-deps.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# The views and conclusions contained in the software and documentation are those
+# of the authors and should not be interpreted as representing official policies,
+# either expressed or implied, of the FreeBSD Project.
+
+usage() {
+  echo "Usage: $0 [--options]"
+  echo "Options:"
+  echo "--dev: install developer libraries"
```
Comment (szledan) [inline tools/scripts/install-build-deps.sh]: Please use `-d` to sort and `--developer` to long version.

**Comment by dati91** [comment]
@dbatyai r?

File: tools/scripts/install-build-deps.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Copyright (C) 2015-2016, Gepard Graphics
```
Comment (dbatyai) [inline tools/scripts/install-build-deps.sh]: Please change this to `2016` only.

**Comment by dbatyai** [comment]
In the current state, it would be a little bit overkill to use python. I think it's okay to use shell scripts for the smaller issues, as long as we keep the scripts short, and switch to python only if they get more complicated.

**Comment by dbatyai** [comment]
LGTM after the copyright is fixed.

File: tools/scripts/install-build-deps.sh [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# The views and conclusions contained in the software and documentation are those
+# of the authors and should not be interpreted as representing official policies,
+# either expressed or implied, of the FreeBSD Project.
+
+usage() {
+  echo "Usage: $0 [--options]"
```
Comment (szledan) [inline tools/scripts/install-build-deps.sh]: Please replace `--options` to `[OPTIONS]`

**Comment by szledan** [comment]
+1, lgtm

File: tools/scripts/install-build-deps.sh [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+# ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# The views and conclusions contained in the software and documentation are those
+# of the authors and should not be interpreted as representing official policies,
+# either expressed or implied, of the FreeBSD Project.
```
Comment (elecro) [inline tools/scripts/install-build-deps.sh]: FreeBSD project? Do we need this block?

Comment (szledan) [inline tools/scripts/install-build-deps.sh]: You are right. But I think it isn't related to this patch, so I am making a fix for it in another PR.

**Comment by szledan** [comment]
lgtm

**Pull request merged**